app/models/vita_partner.rb

```diff
@@ -0,0 +1,18 @@
+# == Schema Information
+#
+# Table name: vita_partners
+#
+#  id                      :bigint           not null, primary key
+#  display_name            :string
+#  drop_off_code           :string
+#  logo_url                :string
+#  name                    :string           not null
+#  referral_code           :string
```

Comment:
might be helpful to rename this to `source_parameter_prefix` to make it consistent with other language throughout the code.